src/content/ui-components/installation/next.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Next.js
+meta:
+  title: Next.js | Tiptap UI Component
+  description: Install and configure Tiptap UI Components in your Next.js project.
+  category: UI Components
+sidebars:
+  hideSecondary: true
+---
+
+import { Callout } from '@/components/ui/Callout'
+
+### Create a New Next.js Project
+
+Run the `init` command to create a new Next.js project or to setup an existing one:
+
+```bash
+pnpm dlx @tiptap/cli init
+```
+
+Choose between a **Next.js** project or a **Monorepo**.
```

Comment:
What kind of monorepo is this? What framework is it built with Does the monorepo also include a Next.js project?